src/services/api.js

```diff
@@ -9,11 +9,14 @@ import {
 const axiosInstance = () => {
   const token = getLocalStorage(TOKEN_KEY)
   const headers = token ? { Authorization: `Bearer ${token}` } : undefined
-  return axios.create({
+  let params = {
```

Comment:
just fyi .. you can actually make this `const`, it's one of these weird things in JS, that constant objects are not really constants, you can not reassign the whole object, but you can actually add or remove a property to it

Reply:
Interesting thanks :)